library/Data/NBA/Stats.hs

```diff
@@ -65,6 +66,19 @@ getSplitRows ::
     -> IO (Either StatsError [a]) -- ^ The return value: an IO action resulting in an error or split rows.
 getSplitRows path splitName params = Except.runExceptT $ getSplitRowsGeneric path splitName params
 
+splitRows ::
+    (Aeson.FromJSON a)
+    => ResponseBody -- ^ The JSON string containing the splits
+    -> SplitName -- ^ The split name.
+    -> Either StatsError [a] -- ^ The return value: an error or split rows.
+splitRows string splitName =
+  either
```

Comment:
Ideally, when the generic version of this function exists, this will simply invoke it `splitRows = splitRowsGeneric`.